### ISSUE TITLE
Throw / log error if pointed toward a non V3 OAPI spec file

### ISSUE THREAD

**Issue by addshore** [enzonotario/vitepress-theme-openapi#9]
Currently you just end up with a broken compoenent and missing details.
Liekly easy fix for early users would be to just log specifically that V3 is the only thing cupported if someone loads V2 :)

**Comment by enzonotario** [comment]
For this we also need an external Parser/Validator that just tell use if its v3 or not.

**Comment by addshore** [comment]
So per the spec

https://swagger.io/specification/

> An OpenAPI document compatible with OAS 3.*.* contains a required [openapi](https://swagger.io/specification/#oas-version) field which designates the version of the OAS that it uses.

So the top level field name `openapi` must be set, and...

> be the [version number](https://swagger.io/specification/#versions) of the OpenAPI Specification that the OpenAPI document uses.

TLDR: you could probably just check for `3.` as the start of this string as an initial check
(you could also do semver parsing if you wanted to)
